pysrc/main.py

```diff
@@ -0,0 +1,32 @@
+"""Main Entry point for app_hash sync"""
+import asyncio
+import os
+
+from dotenv import load_dotenv
+
+from pysrc.fetch.dune import DuneFetcher
+
+
+async def main(dune: DuneFetcher) -> None:
+    """App Hash Sync Logic"""
+    # TODO - use runtime config values here:
+    fname, column = "sync_block", "last_synced_block"
+    block_range = await dune.app_hash_block_range(fname, column)
```

Comment:
I have zero clue about async Python.

But in rust, the await command makes the function wait and it would execute one after another.

Reply:
This needs to be recovered before the next query because the values is used as a parameter in the next query. 

There is nothing here yet that needs to be async at all. I just wanted to make it possible for later.